### DISCUSSION TITLE
Tuxemon encountered any predicaments?

### DISCUSSION THREAD

**Discussion by Davidliang11292** [Tuxemon/Tuxemon#2323] [General]
I've noticed that Tuxemon hasn't been updated for quite some time, and some issues haven't received responses in a while. Has something happened? I understand that every contributor has their own life, family, and more important things to attend to. I'm just a bit curious whether Tuxemon is facing any predicaments or other issues. Of course, I hope it's just that someone is tired. If anyone is tired, please take good care and rest; give me a brief response, or not. Regardless, all of you are amazing, and I am here expressing my gratitude to all the Tuxemon contributors. Wishing you smooth sailing in life and pleasant days ahead!

**Comment by Sanglorian** [comment]
Hi @Davidliang11292 , speaking personally I've been focusing on other projects for a while. I feel like Tuxemon is actually very close to being a finished product that we could share and promote widely, but there's a few sticking points - the wiki being broken, and still no Windows builds to my knowledge. 
Unfortunately those are things beyond my expertise.
That said, there's lots of other interesting things we could do with Tuxemon and I've actually been tinkering away at one of them - a board game where you battle tuxemon! There's already a few 3d models of tuxemon and I've been thinking of commissioning more.

**Reply by ultidonki** [reply]
A board game sounds nice, especially if you could get 3d printed models?! 

I was thinking of creating some Tuxemon illustrations that could be used as cards in a playing game. But, I don't actually play card games so I don't know what the rules or card design would be like. I just like the illustrations sometimes :laughing:

**Comment by Davidliang11292** [comment]
Wow! Looking forward to your Tuxemon spin-off game! But as far as I know, a fix for #2318, a major Bug that causes the game to crash, has not yet been incorporated into the development branch

**Comment by ultidonki** [comment]
I've also been busy with a non-Tuxemon project, so I haven't been doing much. (I only created my new Tuxemon comic as I had the idea and wanted to do it, I'm not going to do more until I have another idea, unfortunately)

> But as far as I know, a fix for https://github.com/Tuxemon/Tuxemon/issues/2318, a major Bug that causes the game to crash, has not yet been incorporated into the development branch

It's true - however, I still haven't tested routes 3 until the end of the game. I wanted to test the rest of the game and get those fixes in first. (I might have some more time soon, so I might create some new issues if I find any more bugs)

Once I've tested the rest of the game, and there are pull requests to fix all the issues, I might ping all the maintainers to get them accepted and make a new version, which I think auto-creates new versions for Flathub too. :smile:  But, until everything's been fixed and tested, I don't think we should create a new version, as there's no point until all the bugs are fixed. In my opinion :smiley:

**Reply by Sanglorian** [reply]
> Once I've tested the rest of the game, and there are pull requests to fix all the issues, I might ping all the maintainers to get them accepted and make a new version, which I think auto-creates new versions for Flathub too. 😄 But, until everything's been fixed and tested, I don't think we should create a new version, as there's no point until all the bugs are fixed. In my opinion 😃

Agreed! But I'm excited to hear how your testing goes when you do get the chance. No pressure of course - we're all volunteers here after all.

**Reply by ultidonki** [reply]
>  But I'm excited to hear how your testing goes when you do get the chance. 

I haven't done any testing in the past few weeks, so thanks for giving me a push, I'll try and prioritize it now my comic and some other stuff has finished :smiley:

**Comment by Davidliang11292** [comment]
I'm looking forward to the new version!

**Comment by HippasusTwo** [comment]
A family member died so I haven't been around at all the past month. I also slowed down naturally because while I can program, embarrassingly my ADHD and doing things in a specific formatting style often don't get along. It was probably my style errors that contributed to some bugs not getting immediately patched despite my giving a solution, although, that doesn't explain the bug you pointed out. That probably has more to do with people not having time to play through the whole game to test it thoroughly.
On the plus side, my lack of consistent focus makes me a jack of all trades, so I can always just go back to doing artwork.

**Reply by Davidliang11292** [reply]
I'm very sorry to hear this news. Please accept my condolences and take care.

**Reply by Sanglorian** [reply]
I'm sorry to hear the news too. Thinking of you.
